Makefile

```diff
@@ -2,7 +2,6 @@
 PROJECT_NAME := provider-helm
 PROJECT_REPO := github.com/crossplane-contrib/$(PROJECT_NAME)
 
-PLATFORMS ?= linux_amd64 linux_arm64
```

Comment:
@muvaf does this need to be dropped?

Reply:
I didn't see it in provider-aws and thought we probably don't need it here either. Is there a reason to keep it? The default in submodule seems to include darwin and windows as well.

Reply:
I believe that it is present in all providers (example in [provider-aws](https://github.com/crossplane-contrib/provider-aws/blob/88bf9b6cd0545c392d3aaa247c074b9b2891d3f6/Makefile#L7)). The only issue it would cause is in image construction, but it looks like it is being skipped correctly. I think building the extra binaries here is not really needed, but I'm not strongly against it.

Reply:
Oh I missed that. I'd prefer not to deviate, reverted that part. Thanks!